### PR TITLE
Crash on iOS 11 during RCTWKWebViewManager destruction

### PR DESCRIPTION
**Pull request by diegum**
I basically let RCTWKWebViewManager stop holding a reference to WKProcessPool. Just left the pool bound to the WKWebView.

### CODE REVIEW

File: ios/RCTWKWebView/RCTWKWebViewManager.m [hunk 1]
```diff
@@ -17,24 +17,19 @@ @implementation RCTWKWebViewManager
 {
   NSConditionLock *_shouldStartLoadLock;
   BOOL _shouldStartLoad;
-  WKProcessPool *_processPool;
```
Comment (diegum) [inline ios/RCTWKWebView/RCTWKWebViewManager.m]: We don't seem to need a hold to this ```WKProcessPool``` reference, as we are passing it to the ```RCTWKWebView``` instance created on ```- (UIView *)view```.

The issue with holding the instance as we are, is that if the app is restarting and all components are destroyed and re-created, the destruction of this ```RCTWKWebViewManager``` instance crashes on iOS 11 when this ```WKProcessPool``` gets disposed.

I don't know if the fact that an ```RCTWKWebView``` is holding a reference could be affecting it at all.

**Comment by gameboyVito** [comment]
Excellent! I have tested this and it works on xcode9 beta6

Please merge and publish a new release, thanks.

**Comment by insraq** [comment]
1.8.0 on npm. Thanks for your PR.

**Comment by Obooman** [comment]
@insraq 

Reload On iOS 11 simulator can cause a crash.
screen shot 2017-09-24 at 2 39 28 pm

Software | Verison
---|---
Xcode |  9.0 (9A235)
simulator | Version 10.0
iOS | 11.0(15A372)
react-native | 0.45.1
react-native-wkwebview-reborn | 0.18.0

**Comment by diegum** [comment]
@Obooman , you seem to be running on a previous version rather than the version this pull request got merged.

I see in your screenshot that the line that reads ``WKProcessPool *_processPool;`` is still there.

If you check above in this pull request, I eliminated that line and explained why. Please get the latest stable branch.

**Comment by Obooman** [comment]
@diegum Thanks for your response and you are right. 

I have checked `node_modules/react-native-wkwebview-reborn/package.json` version, it's `0.16.0` but I have change `package.json` at my root folder to `react-native-wkwebview-reborn:^0.18.0`. It doesn't work. 

So if anyone what to upgrade this package from `0.16.0` to `0.18.0` have to run 
`yarn add react-native-wkwebview-reborn ` or 
`npm i --save react-native-wkwebview-reborn`

**Comment by diegum** [comment]
2 questions:

1. Where do you get ``0.18.0`` from? 
1. How about ``1.8.0`` (which is the tag of this and other recent changes)?

**Comment by Obooman** [comment]
Sorry, I `write it wrong` as I change it right, I'll modify my response.

**Comment by scisci** [comment]
Hi this issue appears to be back in version 1.17.0, I can see `WKProcessPool *_processPool;` and am getting a crash on reload. @diegum was the _processPool reintroduced?

**Pull request merged**